src/Components/Movies.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from "react";
+import APICalls from "./APICalls";
+
+class Movies extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      selectedValue: "",
+      currentMovie: {},
+    };
+  }
+
+
+  handleChange = async (e) => {
+    const {movies} = this.state;
+    this.setState({
+        selectedValue: e.target.value,
+    });
+
+    let movieID = "";
+
```

Comment:
For a really neat shortcut, check out JavaScript's built-in `.find` method on arrays. It simply returns the element that matches your callback. Super useful!